### ISSUE TITLE
Table Block - Table Border will not turn off

### ISSUE THREAD

**Issue by syhussaini** [Automattic/wp-calypso#75198]
### Quick summary

The border of the table block keeps showing even when the border settings is set to zero PX.

![image](https://user-images.githubusercontent.com/8070771/229367386-87bfb774-3483-4a1a-aa98-d473e2cf6524.png)

### Steps to reproduce

1. Insert a Table block in a Page/Post
2. Set the border value to zero PX
3. Preview the Page/Post

![image](https://user-images.githubusercontent.com/8070771/229367445-bf46ebe8-6747-43f8-b23c-1111a43d52a9.png)

### What you expected to happen

I expected the border to disappear when set to zero PX.

### What actually happened

The border keeps appearing.

### Impact

Some (< 50%)

### Available workarounds?

Yes, easy to implement

_Update from cuemarie - adding CSS workaround for now:_

```
.wp-block-table td {
    border: none;
}
```

### Platform (Simple and/or Atomic)

Simple, Atomic

**Comment by susanjanec** [comment]
This issue stemmed from this chat: 41846915-hc

**Comment by cuemarie** [comment]
📌 SCRUBBING : RESULT - *Replicated / Could Not Replicate / Uncertain*
- Tested on Simple – Replicated
- Tested on Atomic – Replicated
- Replicable outside of Dotcom – Yes

📌 FINDINGS/SCREENSHOTS/VIDEO
- Replicated on Simple, Atomic, and self-hosted site with Gutenberg 15.4 activated
     - _deactivating Gutenberg allows the borders to be hidden as expected_

**Screenshots/Recordings**
![Markup on 2023-04-04 at 15:35:02](https://user-images.githubusercontent.com/27249804/229937573-c72f713c-311d-48f2-9b28-9e7223565f22.png)

![Markup on 2023-04-04 at 15:33:51](https://user-images.githubusercontent.com/27249804/229937588-f101ce5f-1afb-4688-ae98-2b8cf4f378bc.png)

Selecting a specific cell in the inspector (`<td>`) reveals the following CSS:

```
.wp-block-table td, .wp-block-table th {
  border: 1px solid;
  padding: .5em;
}
```

---

With Gutenberg 15.4 deactivated on my Pressable site, the table borders respond as expected:

![Markup on 2023-04-04 at 15:48:51](https://user-images.githubusercontent.com/27249804/229939178-b27fd8fc-a84f-443c-a43b-e039b57ef6d4.png)

![Markup on 2023-04-04 at 15:49:17](https://user-images.githubusercontent.com/27249804/229939188-aa4d8a10-f198-48f2-9afb-10437f98ffe9.png)


📌 ACTIONS
- Assigned to Apex
- Added to ‘HE Cross-repo Watchlist’ project board
- Reported in Gutenberg GH Repo: https://github.com/WordPress/gutenberg/issues/49594

**Comment by cuemarie** [comment]
Should be addressed in Gutenberg 15.6: https://github.com/WordPress/gutenberg/pull/49603

**Comment by annezazu** [comment]
Closing this out as the previous Gutenberg issue was merged and this problem resolved upstream: https://github.com/WordPress/gutenberg/pull/49603
